### ISSUE TITLE
Since commit 552d3833 the 'api' folder is missing, yet all the documentation still refers to it

### ISSUE THREAD

**Issue by denhamcoote** [Hygieia/Hygieia#3084]
This is making it rather difficult to install, as I'm not an expert in mvn, nmp, mongo or any of the tools you require.

Could you provide (or point me to) some updated documentation, please?

**Comment by rvema** [comment]
This is a duplicate of #3074 . please check the issue

**Comment by denhamcoote** [comment]
@rvema Thanks for the prompt response, but closing the issue and redirecting doesn't exactly address the issue I raised.

I found the new API repo, that is not the issue.  The issue is the lack of documentation/ incorrect documentation.  Should I raise a new issue but word the title differently? "Documentation incorrect", perhaps?

I'd really like to use Hygieia, but it's really difficult to get started without guidance ;)
